app/build.gradle

```diff
@@ -190,5 +197,12 @@ dependencies {
     implementation("com.google.dagger:hilt-android:2.44")
     kapt("com.google.dagger:hilt-android-compiler:2.47")
 
+    // Jetpack Compose
+    implementation "androidx.compose.material:material:1.3.0-rc01"
```

Comment:
Add the versions in the `ext` block in project level gradle fine for consistency.